### PR TITLE
build: add HOST_CXXFLAGS for host build

### PR DESCRIPTION
**Pull request by svlobanov**
Added HOST_CXXFLAGS to specify CXXFLAGS during host-compile
(e.g. to specify c++ standard: HOST_CXXFLAGS += -std=c++11)

Signed-off-by: Sergey V. Lobanov <sergey@lobanov.in>

### CODE REVIEW

**Comment by aparcar** [comment]
Do you have a specific use case for a package?

**Comment by svlobanov** [comment]
@aparcar yes, it is here https://github.com/openwrt/packages/pull/17618

File: include/host-build.mk [hunk 1]
```diff
@@ -52,6 +52,7 @@ HOST_CONFIGURE_VARS = \
 	CXX="$(HOSTCXX)" \
 	CPPFLAGS="$(HOST_CPPFLAGS)" \
 	LDFLAGS="$(HOST_LDFLAGS)" \
+	CXXFLAGS="$(HOST_CXXFLAGS)" \
```
Comment (hauke) [inline include/host-build.mk:55]: Please put this after the CPPFLAGS, same for the other change

**Review by hauke — APPROVED** [review]

File: include/host-build.mk (hunk 1 above)
Comment (svlobanov) [reply include/host-build.mk:54]: done

**Comment by hauke** [comment]
Thank you for your patch, I applied it to master in 87d489f67a35968242cd4f192dc363dc87056844.

**Pull request closed without merging**